crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -455,18 +455,20 @@ impl<'a> Submitter<'a> {
 
             match self.submit_api.submit_transaction(method.tx).await {
                 Ok(handle) => {
-                    tracing::info!(
+                    tracing::debug!(
                         submitter = %submitter_name, ?handle,
                         "submitted transaction",
                     );
                     transactions.push((handle, gas_price));
                 }
-                Err(err) => tracing::warn!("submission failed: {:?}", err),
+                Err(err) => {
+                    tracing::warn!(
+                        submitter = %submitter_name, ?err,
+                        "submission failed",
+                    );
+                }
             }
-            tracing::debug!(
-                "Finished sending transaction with submitter {}...",
-                submitter_name
-            );
+            tracing::debug!("Finished sending transaction with submitter {submitter_name}...");
```

Comment:
Since both of the logs in the `match` block already contain the `submitter_name` this log no longer adds anything. I think it can be dropped.